### ISSUE TITLE
Add music and sound effects

### ISSUE THREAD

**Issue by Graphcalibur** [Invincibot/covid-tower-defense#200]
Use this thread to request music and sound effects to be added.

**Comment by Graphcalibur** [comment]
Places that need music:
- Level select screen
- Tower select menu
- Upgrades menu
- Game

Some of these could be probably have the same music.

Things that could use sound effects:
- Clicking on buttons
- Selecting/deselecting towers in the Tower Select Menu
- Buzzing noise when trying to select a tower in the UI or upgrade a tower despite not having the required protein
- Text scrolling in the textbox

**Comment by Graphcalibur** [comment]
Went scouring through Newgrounds for some 8-bit music and found these. What do you think @Invincibot?
- https://www.newgrounds.com/audio/listen/643418 - Contrary to the description/title, I think this would work well as the main Menu music (level select, tower select, and upgrades). It's pretty low-key and not too fast-paced.
- https://www.newgrounds.com/audio/listen/603041 - Could work for the first few levels on Mild difficulty. Might be too light-hearted for the tone?
- https://www.newgrounds.com/audio/listen/643430 - Fast but still pretty light-hearted. Would be good for the later levels on Mild/Acute difficulty.
- https://www.newgrounds.com/audio/listen/525911 - Faster-paced and darker with higher stakes. I think this would fit the earlier maps on Acute/Severe.
- https://www.newgrounds.com/audio/listen/928272 - Probably the darkest song in this list. Would be great for the harder/darker levels, like the later maps on Acute/Severe.

Some other songs that I don't think really fit but you might like them:
- https://www.newgrounds.com/audio/listen/760401
- https://www.newgrounds.com/audio/listen/572129

**Comment by vincetiu8** [comment]
I agree with you for pretty much all of them, but i think the last track is a bit too slow/unintense for our game. We could replace it with the last track instead?

**Comment by vincetiu8** [comment]
Oh, and make sure to reference everything in the README.md in the data folder @Graphcalibur.

**Comment by Graphcalibur** [comment]
> I agree with you for pretty much all of them, but i think the **last track** is a bit too slow/unintense for our game. We could replace it with the **last track** instead?

Which tracks are you referring to for these? @Invincibot

**Comment by Graphcalibur** [comment]
Also, a more bombastic win jingle would be nice. The beeps are pretty anticlimactic.

**Comment by vincetiu8** [comment]
Ah yes. Sorry i thought i replied. What I mean is that 
> https://www.newgrounds.com/audio/listen/928272 - Probably the darkest song in this list. Would be great for the harder/darker levels, like the later maps on Acute/Severe.  

Should be replaced with

>https://www.newgrounds.com/audio/listen/572129

**Comment by Graphcalibur** [comment]
Hmm, since we have about 14 maps now, I was thinking of having different songs for the bronchus, heart, and brain maps compared to the other maps. The songs for these maps would have a much more epic/last stand feel compared to the other maps. @Invincibot 

The songs I had in mind:
- Mild - https://www.newgrounds.com/audio/listen/923539
- Acute - https://www.newgrounds.com/audio/listen/336068
- Severe - https://www.newgrounds.com/audio/listen/367084

**Comment by vincetiu8** [comment]
They sound good to me!
